logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationGroupRepository.kt

```diff
@@ -271,4 +272,11 @@ internal class ConversationGroupRepositoryImpl(
 
     override suspend fun observeGuestRoomLink(conversationId: ConversationId): Flow<String?> =
         conversationDAO.observeGuestRoomLinkByConversationId(conversationId.toDao())
+
+    override suspend fun updateMessageTimer(conversationId: ConversationId, messageTimer: Long?): Either<NetworkFailure, Unit> =
```

Comment:
I think messageTimer does not explain the purpose of this well enough, for someone outside the context